src/echo-app/src/renderer/src/plugin-page.tsx

```diff
@@ -6,6 +6,7 @@ import { EchoRoute } from 'echo-common/dist/cjs/echo-router'
 import React, { useEffect } from 'react'
 import { ProfilePage } from './profile-page'
 import { PluginSettingsPage } from './plugin-settings-page'
+// @ts-ignore
 import { DEV_PLUGINS } from './dev-plugins'
 
```

Comment:
This is fine if there is no `./dev-plugins` right?

Reply:
I deleted this file and added it to .gitignore.

For the import i use //ts-ignore